hooks/command

```diff
@@ -1,589 +1,16 @@
-  fi
-fi
-
-if [[ "${BUILDKITE_PLUGIN_DOCKER_EXPAND_IMAGE_VARS:-false}" =~ ^(true|on|1)$ ]] ; then
-  image=$(eval echo "${BUILDKITE_PLUGIN_DOCKER_IMAGE}")
-else
-  image="${BUILDKITE_PLUGIN_DOCKER_IMAGE}"
-fi
-
-if [[ "${BUILDKITE_PLUGIN_DOCKER_ALWAYS_PULL:-false}" =~ ^(true|on|1)$ ]] ; then
-  echo "--- :docker: Pulling ${image}"
-  if ! retry "${BUILDKITE_PLUGIN_DOCKER_PULL_RETRIES:-3}" \
-       docker pull "${image}" ; then
-    echo "!!! :docker: Pull failed."
-    exit "$retry_exit_status"
-  fi
-fi
-
-# Parse network and create it if it don't exist.
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_NETWORK:-}" ]] ; then
-  DOCKER_NETWORK_ID=$(docker network ls --quiet --filter "name=${BUILDKITE_PLUGIN_DOCKER_NETWORK}")
-  if [[ -z ${DOCKER_NETWORK_ID} ]] ; then
-    echo "creating network ${BUILDKITE_PLUGIN_DOCKER_NETWORK}"
-    docker network create "${BUILDKITE_PLUGIN_DOCKER_NETWORK}"
-  else
-    echo "docker network ${BUILDKITE_PLUGIN_DOCKER_NETWORK} already exists"
-  fi
-  args+=("--network" "${BUILDKITE_PLUGIN_DOCKER_NETWORK:-}")
-fi
-
-# Support docker run --platform
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_PLATFORM:-}" ]] ; then
-  args+=("--platform" "${BUILDKITE_PLUGIN_DOCKER_PLATFORM:-}")
-fi
-
-# Support docker run --pid
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_PID:-}" ]] ; then
-  args+=("--pid" "${BUILDKITE_PLUGIN_DOCKER_PID:-}")
-fi
-
-# Support docker run --gpus
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_GPUS:-}" ]] ; then
-  args+=("--gpus" "${BUILDKITE_PLUGIN_DOCKER_GPUS:-}")
-fi
-
-# Support docker run --runtime
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_RUNTIME:-}" ]] ; then
-  args+=("--runtime" "${BUILDKITE_PLUGIN_DOCKER_RUNTIME:-}")
-fi
-
-# Support docker run --ipc
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_IPC:-}" ]] ; then
-  args+=("--ipc" "${BUILDKITE_PLUGIN_DOCKER_IPC:-}")
-fi
-
-# Support docker run --storage-opt
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_STORAGE_OPT:-}" ]] ; then
-  args+=("--storage-opt" "${BUILDKITE_PLUGIN_DOCKER_STORAGE_OPT:-}")
-fi
-
-shell=()
-shell_disabled=1
-
-if [[ -n "${BUILDKITE_COMMAND}" ]]; then
-  if [[ $(echo "$BUILDKITE_COMMAND" | wc -l) -gt 1 ]]; then
-    # An array of commands in the step will be a single string with multiple lines
-    # This breaks a lot of things here so we will print a warning for user to be aware
-    echo "⚠️  Warning: The command received has multiple lines."
-    echo "⚠️           The Docker Plugin may not correctly run multiple commands in the step-level configuration."
-    echo "⚠️           You will need to use a single command, a script or the plugin's command option."
-  fi
-  shell_disabled=''
-fi
-
-# Handle setting of shm size if provided
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_SHM_SIZE:-}" ]]; then
-  args+=("--shm-size" "${BUILDKITE_PLUGIN_DOCKER_SHM_SIZE}")
-fi
-
-# Handle setting of cpus if provided
-if [[ -n "${BUILDKITE_PLUGIN_DOCKER_CPUS:-}" ]]; then
-  args+=("--cpus=${BUILDKITE_PLUGIN_DOCKER_CPUS}")
+if [[ -n "$(plugin_read_list LOAD)" ]]; then
```

Comment:
Here the same, as the `load` option is just a string, you should just instead use:

```suggestion
if [[ -n "${BUILDKITE_PLUGIN_DOCKER_LOAD:-}" ]]; then
```